test/test012.md

```diff
@@ -0,0 +1,42 @@
+Valid entries:
+1 January 1970
+14 July 2023
+
+30 invalid entries:
+
+day-month-year
+1/1/70
+01/11/1970
+1/11/70
+01/01/70
+31/1/1973
+02/1/1973
+31.1.2011
+31-1-2001
+29/2/1973
+29/02/1976 
+03/06/2010
+12/6/90
+
+month-day-year
+02/24/1975 
+06/19/66 
+03.31.1991
+2.29.2003
+02-29-55
+03-13-55
+03-13-1955
+12\24\1974
+12\30\1974
+1\31\1974
+03/31/2001
+01/21/2001
+12/13/2001
+
+year-month-day
+76/02/02
+1976/02/29
+1976/2/13
+76/09/31
```

Comment:
Should be covered, but maybe add something like `2024-07-16` - that would be my typical way of writing a date. ;)